app/models/flood_risk_engine/reference_number.rb

```diff
@@ -0,0 +1,28 @@
+module FloodRiskEngine
+  class ReferenceNumber < ActiveRecord::Base
+    PREFIX = "EXFRA".freeze
+    PADDING = "0".freeze
+    OFFSET = 10_000
```

Comment:
If you look at the original spec., the offset was given by Polly as 10,000. Not problem changing it to 1000.